app/pages/search/Sort.js

```diff
@@ -0,0 +1,65 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { injectT } from 'i18n';
+import SelectControl from './controls/SelectControl';
+
+class Sort extends Component {
+  constructor(props) {
+    super();
+    this.state = {
+      selected: '',
+    };
+
+    const { lang } = props;
+    this.sortOptions = [
+      { label: 'Name', value: `resource_name_${lang}` },
+      { label: 'Type', value: 'type' },
+      { label: 'Premises', value: `unit_name_${lang}` },
+      { label: 'People', value: 'people_capacity' },
+      { label: 'Open now', value: 'sortByOpenNow' },
+    ];
+  }
+
+  handleChange = ({ value }) => {
+    const filters = {};
+    if (value === 'sortByOpenNow') {
+      const now = (new Date()).toISOString();
+      filters.orderBy = null;
+      filters.available_between = `${now},${now}`;
```

Comment:
{now} {now} ?

Reply:
There is this `available_between` in API documentation which accepts / requires 2 values. It is just comparing the current time to check the currently opened resource. So, these two `now` is  a trick to get the intended result of open now resource as `available_between accepts 2 values`.

Reply:
I discussed with backend guy, make the `endTime` as the end of the day. Next time, don't do trick.

Reply:
End of the day added in the new commit as suggested. But I am not receiving the data from API, if you could have a look.

Reply:
@Rasbin: No idea, let me know only when PR is ready to review.

Reply:
Sort By open now removed in the recent commit due to no API support.